django/channels/asgi.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import unicode_literals
+
+import django
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+from .routing import Router
+from .utils import name_that_thing
+
+
+class InvalidChannelLayerError(ValueError):
+    pass
+
+
+class ChannelLayerManager(object):
+    """
+    Takes a settings dictionary of backends and initialises them on request.
```

Comment:
For new code, we're using PEP257 verb style "Take....", 
initialises -> initializes